JavaScript/packages/recognizers-date-time/src/dateTime/chinese/dateTimeConfiguration.ts

```diff
@@ -349,4 +389,84 @@ export class ChineseDateTimeParser extends BaseDateTimeParser {
 
         return ret;
     }
+
+    // Handle cases like "5分钟前", "1小时以后"
+    private parserDurationWithBeforeAndAfter(source: string, referenceDate: Date): DateTimeResolutionResult {
```

Comment:
Is this just the same repetition of this method?
We should look into unifying implementations. Maybe as an util like in .NET?